minitron.rb

```diff
@@ -51,6 +51,8 @@
       timestamp: ticket.created_at,
       properties: {
         ticket_id: id,
+        requester_name: ticket.requester.name,
+        requester_org: ticket.requester.organization.name,
```

Comment:
Same here. This should come through via `group` call, not as part of the event properties.